server/data/visitSchedulerApiClient.ts

```diff
@@ -144,4 +158,19 @@ class VisitSchedulerApiClient {
   }
 }
 
+function convertMainContactToVisitContact(mainContact: VisitSessionData['mainContact']): {
```

Comment:
Not sure if this would be better as a method inside the class? Just keeps it tidy?

Reply:
Good spot! Have tidied this up 🙂

Reply:
Approved 🙂